Sample/SampleForageSDK/Foundation/Network/Model/ForagePANModel.swift

```diff
@@ -21,10 +21,12 @@ public struct ForagePANModel: Codable {
     public let paymentMethodIdentifier: String
     public let type: String
     public let card: ForageCard
+    public let customerID: String
     
     private enum CodingKeys : String, CodingKey {
         case paymentMethodIdentifier = "ref"
         case type
         case card
+        case customerID = "user_id"
```

Comment:
I'm wondering if this needs to be changed like this?

```suggestion
        case customerID = "customer_id"
```

Reply:
This whole file is being deleted in a different PR since the code is unused. However, I did just remove it in a commit to be complete.